analytics/sam/cicd/template.yaml

```diff
@@ -26,7 +26,7 @@ Resources:
     Properties:
       Location:
         ApplicationId: arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd
-        SemanticVersion: 0.1.3
+        SemanticVersion: 0.1.4
```

Comment:
Why this change? That semver doesn't exist... You should only be bumping semvers in the template metadata section as well as any `AWS::Serverless::Application`s that reference apps whose MVs you're bumping in this PR. Same feedback for all places where you're bumping the version on aws-sam-codepipeline-cd.

```text
% aws serverlessrepo list-application-versions --application-id arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd
{
    "Versions": [
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-09-04T23:50:57.809Z",
            "SemanticVersion": "0.1.0",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/0.1.0"
        },
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-09-04T23:54:00.997Z",
            "SemanticVersion": "0.1.1",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/0.1.1"
        },
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-09-16T19:26:49.995Z",
            "SemanticVersion": "0.1.3",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/0.1.3"
        },
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-10-14T01:17:47.607Z",
            "SemanticVersion": "1.0.0",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/1.0.0"
        },
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-10-25T19:11:46.577Z",
            "SemanticVersion": "1.1.0",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/1.1.0"
        }
    ]
}
```